source/manual/development-pipeline.html.md

```diff
@@ -14,83 +14,135 @@ review_in: 6 months
+> If you need to freeze deployments for an application, you should
+> also make people aware using other channels.
 
 ## Merge your own Pull Request
 
 We've got [guidelines on merging of Pull Requests](/manual/merge-pr.html).
 
-> Some applications [have restrictions on who can merge PRs](https://github.com/alphagov/govuk-saas-config/blob/master/github/repo_overrides.yml). If you are unable to merge your own PR, you should ask [someone else](https://github.com/orgs/alphagov/teams/gov-uk-production/members) to merge (and deploy) it for you.
+Some applications [have restrictions on who can merge PRs](https://github.com/alphagov/govuk-saas-config/blob/master/github/repo_overrides.yml).
+If you are unable to merge your own PR, you should ask
+[someone else](https://github.com/orgs/alphagov/teams/gov-uk-production/members)
+to merge (and deploy) it for you.
+
+Code that is merged to `master` is tested again on CI. This is because
+the `master` branch may have changed since the tests last ran on the PR.
+If the tests on `master` pass, Jenkins pushes a `release_123` git tag to
+GitHub.
+
+> **WARNING**: some applications have Continuous Deployment enabled,
+> which means the deployment process is fully automated. You should do
+> any manual testing with [a temporary, branch deployment](#branch-deploy-review)
+> before you merge.
+
+## Deployment
 
-Code that is merged to `master` is tested again on CI. This is because the `master` branch may have changed since the tests last ran on the PR. If the tests on `master` pass, Jenkins pushes a `release_123` git tag to GitHub.
+After merging your PR, the application now needs deploying.
 
-> **WARNING**: some applications have Continuous Deployment enabled, which means the deployment process is fully automated. You should do any manual testing with [a temporary, branch deployment](#branch-deploy-review), before you merge.
+### Continuous Deployment
 
 - Check the notes in the [Release app][release] to see if Continuous Deployment is enabled.
 - If so, after merging, you should check the Release app to see if the deployment succeeds.
 - If the latest release is not on Production within about 15 minutes, something went wrong:
   - Refer to the [Continuous Deployment Demo slides][slides] for details of the deployment process so you can pinpoint where in the pipeline it failed.
   - You can manually deploy your change if the automation fails e.g. due to a flakey [Smokey test][smokey].
 
-## Deployment
+### Manual Deployment
 
-Teams are responsible for deploying their own work. We believe that [regular releases minimise the risk of major problems](https://gds.blog.gov.uk/2012/11/02/regular-releases-reduce-risk) and improve recovery time. The [2nd line team](/manual/welcome-to-2nd-line.html) is responsible for providing access to deploy software for teams who can't deploy it themselves.
+Teams are responsible for deploying their own work. We believe that
```

Comment:
This section pertain to deployment in general. Even with CD, there are still restrictions on who can deploy (/merge) stuff. Can we move it back up?